static/files/forms-data.json

```diff
@@ -264,6 +147,206 @@
+          "id": "hardening-requirements",
+          "fields": [
+            {
+              "type": "checkbox",
+              "id": "pci",
+              "value": "PCI-DSS",
+              "label": "PCI-DSS"
+            },
+            {
+              "type": "checkbox",
+              "id": "hipaa",
+              "value": "HIPAA",
+              "label": "HIPAA"
+            },
+            {
+              "type": "checkbox",
+              "id": "fisma",
+              "value": "FISMA",
+              "label": "FISMA"
+            },
+            {
+              "type": "checkbox",
+              "id": "fips-140",
+              "value": "FIPS 140",
+              "label": "FIPS 140"
+            },
+            {
+              "type": "checkbox",
+              "id": "ncsc",
+              "value": "NCSC",
+              "label": "NCSC"
+            },
+            {
+              "type": "checkbox",
+              "id": "disa-stig",
+              "value": "DISA-STIG",
+              "label": "DISA-STIG"
+            },
+            {
+              "type": "checkbox",
+              "id": "fedramp",
+              "value": "FedRAMP",
+              "label": "FedRAMP"
+            },
+            {
+              "type": "checkbox",
+              "id": "cis-benchmark",
+              "value": "CIS Benchmark",
+              "label": "CIS Benchmark"
+            }
+          ]
+        },
+        {
+          "title": "Who is responsible for tracking, testing and applying CVE patches in a timely manner?",
+          "id": "responsible-for-tracking",
+          "fields": [
+            {
+              "type": "checkbox",
+              "id": "individual-developers",
+              "value": "Individual developers",
+              "label": "Individual developers"
+            },
+            {
+              "type": "checkbox",
+              "id": "project-team",
+              "value": "The project team",
+              "label": "The project team"
+            },
+            {
+              "type": "checkbox",
+              "id": "third-party-vendor",
+              "value": "Third-party vendor",
+              "label": "Third-party vendor"
+            },
+            {
+              "type": "checkbox",
+              "id": "i-dont-know",
+              "value": "I don't know",
+              "label": "I don't know"
+            }
+          ]
+        },
+        {
+          "title": "What advice are you looking for?",
+          "id": "advice",
+          "noCommentsFromLead": true,
+          "fields": [
+            {
+              "type": "long-text",
+              "id": "advice",
+              "label": "Tell us about your challenges and your goals"
+            }
+          ]
+        },
+        {
+          "title": "How should we get in touch?",
+          "id": "about-you",
+          "noCommentsFromLead": true,
+          "fields": [
+            {
+              "type": "text",
```

Comment:
Good idea since most of our forms do require a contact us section, I'll add this feature

Reply:
Good idea since most of our forms do require a contact us section, I've created a [separate ticket](https://warthogs.atlassian.net/browse/WD-17910) for this as it is out of scope for this PR